shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -574,6 +379,91 @@ export default {
   },
 
   methods: {
+    // To display the provider and kubernetes in the masthead of this page we need to retrieve the appropriate node templates so they models can find the data
+    loadNodeTemplateForMasthead() {
+      if (this.$store.getters['management/canList'](MANAGEMENT.NODE_TEMPLATE)) {
+        const args = { id: MANAGEMENT.NODE_TEMPLATE, context: 'provisioning.cattle.io.cluster' };
+
+        if (this.$store.getters[`management/paginationEnabled`]?.(args)) {
+          const templateOpt = PaginationParamFilter.createMultipleFields([{
+            field: 'spec.clusterName',
```

Comment:
this reference, and snapshot and machine deployments need adding to StevePaginationUtils.VALID_FIELDS and then also to https://github.com/rancher/rancher/issues/48603

Reply:
Done for both